# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/opensubdiv/hotfix-cmake.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- OpenSubdiv-3_7_0/opensubdiv-config.cmake.in.vanilla	2026-09-27 16:13:11.509192157 +0300
+++ OpenSubdiv-3_7_0/opensubdiv-config.cmake.in	2026-09-27 16:14:41.764873734 +0300
@@ -1,6 +1,6 @@
 @PACKAGE_INIT@
 
 set_and_check(OpenSubdiv_INCLUDE_DIR "${PACKAGE_PREFIX_DIR}/@CMAKE_INSTALL_INCLUDEDIR@")
-set_and_check(OpenSubdiv_LIB_DIR "${PACKAGE_PREFIX_DIR}/@CMAKE_INSTALL_LIBDIR@")
+set_and_check(OpenSubdiv_LIB_DIR "@CMAKE_INSTALL_LIBDIR@")
 
 include("${CMAKE_CURRENT_LIST_DIR}/OpenSubdivTargets.cmake")
